src/SixLabors.Fonts/Tables/Woff2Utils.cs

```diff
@@ -47,77 +47,102 @@ internal class Woff2Utils
 
         public static Woff2TableHeader Read(BigEndianBinaryReader reader, uint expectedTableStartAt, out uint nextExpectedTableStartAt)
         {
-            byte flags = reader.ReadByte();
-            int knowTable = flags & 0x1F;
-            string tableName = (knowTable < 63) ? KnownTableTags[knowTable] : reader.ReadTag();
-            byte preprocessingTransformation = (byte)((flags >> 5) & 0x3);
+            // Leave the first byte open to store flagByte
+            const uint woff2FlagsTransform = 1 << 8;
+            byte flagsByte = reader.ReadByte();
+            int knownTable = flagsByte & 0x3F;
+            string tableName = knownTable == 0x3F ? reader.ReadTag() : KnownTableTags[knownTable];
 
-            if (!ReadUIntBase128(reader, out uint tableOrigLength))
+            uint flags = 0;
+            byte xformVersion = (byte)((flagsByte >> 6) & 0x03);
+
+            // 0 means xform for glyph/loca, non-0 for others
+            if (tableName is "glyf" or "loca")
```

Comment:
minor: use const for the table names?

Reply:
I _think_ the compiler will just make these constants anyway.